src/pages/Shared/NavBar/NavBar.js

```diff
@@ -0,0 +1,235 @@
+import { AppBar, styled, Toolbar, Typography , InputBase, Badge ,Tab, Tabs, Avatar, Tooltip, Divider,Menu,  MenuItem, IconButton} from '@mui/material';
+import React, {useState} from 'react';
+import blithelogo from '../../../Assets/logo/logo-gray.jpg'
+import './NavBar.css';
+import { TiSocialLinkedinCircular } from "react-icons/ti";
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import MailIcon from '@mui/icons-material/Mail';
+import { Box, fontSize } from '@mui/system';
+import HomeIcon from '@mui/icons-material/Home';
+import WorkIcon from '@mui/icons-material/Work';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import PersonAdd from '@mui/icons-material/PersonAdd';
+import Settings from '@mui/icons-material/Settings';
+import Logout from '@mui/icons-material/Logout';
+import PublicIcon from '@mui/icons-material/Public';
```

Comment:
suggestion: all material related imports can be done in a file and imported here from that file

Reply:
ok